editor/src/components/canvas/dom-walker.spec.browser2.tsx

```diff
@@ -59,6 +59,7 @@ async function renderTestEditorWithCode(appUiJsFileCode: string) {
 
   const initialEditorStore: EditorStore = {
     unpatchedEditor: emptyEditorState,
+    patchedEditor: emptyEditorState,
     editor: emptyEditorState,
```

Comment:
Are we not removing this reference?

Reply:
This is what my note at the end is about - I've kept this because of the `useEditorState` hooks relying on it, but I'm currently looking into how painful it would be to remove it completely from the `EditorStore`